src/plugins.js

```diff
@@ -33,7 +33,8 @@ class Plugins {
         fetch,
         FormData,
         logger,
-        jsonld
+        jsonld,
+        nativeImage
```

Comment:
What's the reason for making some of these part of the context? The plugin can require `nativeImage` if needed, right? How about npm modules included with Tropy, like `jsonld`, would the plugin (the way it's currently set up) be able to load them? `fetch` and `FormData` are already available anyway, so there is little reason to add them to the context is there?

Reply:
The plugin could require `nativeImage` from electron directly, but then it would need an explicit dependency on electron. It's cleaner if there is either an explicit dependency on electron, or these things are passed in with the context. The plugin would currently be able to just `require('jsonld')` when run as a plugin within tropy. But for running in standalone mode, it would need the dependency. I think it makes things more clear when we don't just require things that we think are present in tropy. I have to think about fetch and FormData, maybe you're right...